### ISSUE TITLE
Feature request: use backspace for previous page navigation

### ISSUE THREAD

**Issue by mo6** [zevv/termtekst#16]
- Feature request: use backspace for previous page navigation

**Comment by zevv** [comment]
What's the corresponding "next page' navigation key for that then?

**Comment by Dobefu** [comment]
I think Return would make sense to move to the next page

**Comment by mo6** [comment]
Let me rephrase this in a user story: "As a user, when selecting a page using a 3 digit number when on an overview page (like 501), I want be able to navigate back quick using the backspace key to the page visited previously, so I can select another page from the overview page"

So navigating forward is implied here, hence the request to go back. I would think a call stack (excluding automatic refresh/forwards) would be handy. I'll create a PR for this.

**Comment by zevv** [comment]
Ah, right, I interpreted "backwards" as going to the previous page
number, but your request is about the previous visited page - implying
we keep a (limited) navigation stack and allow traversing the history.

Excellent feature.

--
:wq
^X^Cy^K^X^C^C^C^C
